@ndlib/gatsby-theme-marble/src/components/Internal/ReturnToSearch/index.js

```diff
@@ -3,14 +3,19 @@ import PropTypes from 'prop-types'
 import typy from 'typy'
 import Link from 'components/Internal/Link'
 import style from './style.module.css'
+import { BaseStyles } from 'theme-ui'
+import backToSearch from 'assets/icons/svg/baseline-arrow_back-24px.svg'
 
 export const ReturnToSearch = ({ location }) => {
   if (typy(location, 'state.referal.type').safeString === 'search') {
     return (
       <nav className={style.returnToSearch}>
-        <Link
-          to={`/search${location.state.referal.query}`}
-        >≪ Return to Search</Link>
+        <BaseStyles>
+          <Link
+            to={`/search${location.state.referal.query}`}
+          ><img src={backToSearch} alt='≪ Return to Search' title='Return to Search' />
```

Comment:
So a couple things here:
1. We still want text to appear "Return to search"
2. The image needs to be smaller and match the link color. Take a look at: https://github.com/ndlib/marble-website-starter/blob/master/%40ndlib/gatsby-theme-marble/src/components/Internal/ActionModal/index.js You will need to use `useThemeUI` to get the color and then embed the svg directly in the page. I'd say, let's go about 14px on the width and height.

Here's roughly what your target end result should look like: 
![Screen Shot 2020-05-27 at 9 05 11 AM](https://user-images.githubusercontent.com/416559/83023429-e58bb480-9ffa-11ea-9985-3a85983a57d5.png)

A couple of implementation notes:
From ActionModal:
![Screen Shot 2020-05-27 at 9 10 50 AM](https://user-images.githubusercontent.com/416559/83022759-01428b00-9ffa-11ea-9406-b9570d2ac1fe.png)
Note the `sx` prop on the `svg` that's where you can apply styling. `width` and `height` can be used here to reduce the size of the svg. You probably **do not** want to adjust the `view_box` that is basically the equivalent of adjusting the canvas size in photoshop and changing these values without changing the paths would crop or stretch the svg.
The `fill={fillColor}` on the first `path` in the original svg file there was no fill, when this is the case it defaults to black. On the second path, this is the negative space and `fill='none'`



Note: This will no longer use an `img` tag since it will embed the svg directrly. However, as an FYI this is a decorative image that enhances the text next to it, so it should not need a `title` and `alt` should equal an empty string `''`.